app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/helpers.js

```diff
@@ -31,3 +31,14 @@ export const getVmsWithTargetClusterName = (vms, destinationTenantIdsBySourceClu
       target_cluster_name: tenant ? tenant.name : ''
     };
   });
+
+export const allFitForVM = (bestFitFlavors, tenantFlavors, vmId) => {
+  const flavorsSlugPrefix = 'flavors/';
+  const flavorForVM = bestFitFlavors.find(flavor => flavor.source_href_slug === `vms/${vmId}`);
+  const allFitFlavors = flavorForVM && flavorForVM.all_fit;
+  if (!allFitFlavors || allFitFlavors.length === 0) {
+    return [tenantFlavors.reduce((prev, current) => (prev.root_disk_size > current.root_disk_size ? prev : current))];
```

Comment:
Do we want to switch to memory/cpu here as well?

Reply:
Reading through this one more time, this reads as if

> if there are no all fit flavors, then find the "biggest flavor" and do not mark it with an asterisk

Is this desired?  I think all flavors that are too small should be marked as a warning

Reply:
Ah, that's true. I borrowed this helper code from Aparna's PR here: https://github.com/ManageIQ/manageiq-v2v/pull/655/files, really it should just return nothing if there is no all fit flavor. I'll take a closer look.

Reply:
This should be fixed in 7644f6b.